packages/airnode-examples/integrations/authenticated-coinmarketcap/config.example.json

```diff
@@ -11,7 +11,12 @@
           "url": "${PROVIDER_URL}"
         }
       },
-      "type": "evm"
+      "type": "evm",
+      "chainOptions": {
```

Comment:
Each chain object has a field called `id` and not `chainId` based on the assumption that there is no room for confusion to the degree that `chain` there is redundant (this is apparent when you consider`config.chains[0].id` vs `config.chains[0].chainId`). So based on this, I'd recommend `options` here.

Reply:
That makes sense, I will amend it :)